data/modelCuration/complexAnnotation/complexAnnotation.tsv

```diff
@@ -0,0 +1,46 @@
+r_3218	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_3219	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_3220	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_3221	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_3222	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_3223	YLR386W and YNL325C	YFR019W and YNL325C and YFR021W and YNL054W and YLR386W	https://www.ebi.ac.uk/complexportal/complex/CPX-3088			"YFR019W, YFR021W and YNL054W are also necessary"
+r_1085	YBR127C and YDL185W and YEL027W and YEL051W and YGR020C and YHR026W and YHR039C-A and YKL080W and YLR447C and YMR054W and YOR332W and YPL234C and YPR036W	(YPR170W-B and YBR127C and YDL185W and YOR332W and YHR026W and YEL027W and YKL080W and YLR447C and YEL051W and YPL234C and YMR054W and YGR020C and YPR036W and YHR039C-A and YGR106C and YCL005W-A) or (YBR127C and YDL185W and YOR332W and YHR026W and YEL027W and YKL080W and YLR447C and YEL051W and YPL234C and YMR054W and YGR020C and YPR036W and YHR039C-A and YGR106C and YCL005W-A) 	https://www.ebi.ac.uk/complexportal/complex/CPX-1192			
+r_1086	YBR127C and YDL185W and YEL027W and YEL051W and YGR020C and YHR026W and YHR039C-A and YKL080W and YLR447C and YOR270C and YOR332W and YPL234C and YPR036W	(YBR127C and YDL185W and YEL027W and YEL051W and YGR020C and YHR026W and YHR039C-A and YKL080W and YLR447C and YOR270C and YOR332W and YPL234C and YPR036W and YPR170W-B and YGR106C and YCL005W-A) or (YBR127C and YDL185W and YEL027W and YEL051W and YGR020C and YHR026W and YHR039C-A and YKL080W and YLR447C and YOR270C and YOR332W and YPL234C and YPR036W  and YGR106C and YCL005W-A)	https://www.ebi.ac.uk/complexportal/complex/CPX-1193			
+r_0886	YMR205C or (YGR240C and YMR205C)	YGR240C and YMR205C	https://www.ebi.ac.uk/complexportal/complex/CPX-554	https://www.yeastgenome.org/locus/S000004818	https://www.uniprot.org/uniprot/P16862	YMR205C is a subunit
+r_0887	YMR205C or (YGR240C and YMR205C)	YGR240C and YMR205C	https://www.ebi.ac.uk/complexportal/complex/CPX-554	https://www.yeastgenome.org/locus/S000004818	https://www.uniprot.org/uniprot/P16862	YMR205C is a subunit
+r_2836	YDR373W and YNL267W	(YDR373W and YNL267W) or (YDR373W and YNL267W and YJL100W)			https://www.uniprot.org/uniprot/P42951	"YDR373W, YNL267W, YJL100W could also catalyze the reaction"
```

Comment:
I cannot see the information about the complex for `YDR373W and YNL267W and YJL100W` from [UniProt](https://www.uniprot.org/uniprot/P42951#interaction). I see `YJL100W` can also catalyse this reaction, but not sure wether these three can form a complex.

Reply:
In 4306061 this was now reverted back to `YDR373W and YNL267W`, but if `YJL100W` is a type II phosphatidylinositol 4-kinase ([SGD](https://www.yeastgenome.org/locus/S000003636)), should it not be `(YDR373W and YNL267W) or YJL100W`?

Reply:
Ok, `YJL100W` should not be included when considering that `YDR373W` and `YNL267W` are both essential genes (and not annotated to other reactions (while `YJL100W` is not essential, but this is irrelevant). So how confident is that SGD annotation?